cmd/dbaas_create_grafana.go

```diff
@@ -0,0 +1,76 @@
+package cmd
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/exoscale/cli/pkg/account"
+	"github.com/exoscale/cli/pkg/globalstate"
+	exoapi "github.com/exoscale/egoscale/v2/api"
+	"github.com/exoscale/egoscale/v2/oapi"
+	"github.com/spf13/cobra"
+)
+
+func (c *dbaasServiceCreateCmd) createGrafana(_ *cobra.Command, _ []string) error {
+	var err error
+
+	ctx := exoapi.WithEndpoint(gContext, exoapi.NewReqEndpoint(account.CurrentAccount.Environment, c.Zone))
```

Comment:
AFAIK it's bad practice to use a global context. IMHO we should stop using it in new code, where possible.

Reply:
I agree it is a bad practice but at the moment global context has CTRL+C listener and i can't replace it locally. I'll tackle the issue in the separate effort.